### PR TITLE
Job manager actions view

### PR DESCRIPTION
**Pull request by assafchamoy**

### CODE REVIEW

File: src/discrete-layer/components/system-status/cell-renderer/status.cell-renderer.css [hunk 1]
```diff
@@ -45,11 +45,11 @@
   align-items: center;
   flex-direction: column-reverse;
   position: relative;
-  padding:2px;
+  /* padding:2px; */
```
Comment (alebinson) [inline src/discrete-layer/components/system-status/cell-renderer/status.cell-renderer.css:48]: remove unneeded comment

File: src/discrete-layer/components/system-status/jobs-dialog.css [hunk 2]
```diff
@@ -84,8 +85,22 @@ body[dir=rtl] #jobsDialog .refreshContainer .refreshSecs {
...
 /* Hide border of right-cols-container */
-#jobsDialog .ag-pinned-right-cols-container .ag-cell {
+.jobsBody  .ag-pinned-right-cols-container .ag-row-hover .ag-cell {
+  border: none !important;
+  overflow: visible;
+}
+
+.jobsBody .ag-pinned-right-cols-container .ag-cell {
   border: none !important;
+  width: 20px!important;
```
Comment (alebinson) [inline src/discrete-layer/components/system-status/jobs-dialog.css:95]: add " " space

File: src/discrete-layer/components/system-status/jobs-dialog.css [hunk 3]
```diff
@@ -84,8 +85,22 @@ body[dir=rtl] #jobsDialog .refreshContainer .refreshSecs {
...
+  height: 30px!important;
```
Comment (alebinson) [inline src/discrete-layer/components/system-status/jobs-dialog.css:96]: add " " space

File: src/discrete-layer/components/system-status/jobs-dialog.tsx [hunk 4]
```diff
@@ -79,8 +84,33 @@ export const SystemJobsComponent: React.FC<SystemJobsComponentProps> = observer(
...
+
   const store = useStore();
 
+
+  const getJobActions = useMemo(() => {
+    const actions: IActionGroup[] = store.actionDispatcherStore.getEntityActionGroups(
+      JOB_ENTITY
+    );
+
+    actions[0].group = actions[0].group.map((action) => {
```
Comment (alebinson) [inline src/discrete-layer/components/system-status/jobs-dialog.tsx:96]: why [0]?

File: src/discrete-layer/components/system-status/jobs-dialog.tsx [hunk 5]
```diff
@@ -305,7 +387,7 @@ export const SystemJobsComponent: React.FC<SystemJobsComponentProps> = observer(
             rowData={gridRowData}
             style={{
               height: 'calc(100% - 64px)',
-              width: 'calc(100% - 8px)',
+              // width: 'calc(100% - 100px)',
```
Comment (alebinson) [inline src/discrete-layer/components/system-status/jobs-dialog.tsx:390]: remove unneeded comment

**Review by alebinson — CHANGES_REQUESTED** [review]
Looks good.
There are a few comments(mostly cosmetics)

File: src/discrete-layer/components/system-status/jobs-dialog.css (hunk 2 above)
Comment (assafchamoy) [reply src/discrete-layer/components/system-status/jobs-dialog.css:95]: OK

File: src/discrete-layer/components/system-status/jobs-dialog.css (hunk 3 above)
Comment (assafchamoy) [reply src/discrete-layer/components/system-status/jobs-dialog.css:96]: OK

File: src/discrete-layer/components/system-status/cell-renderer/status.cell-renderer.css (hunk 1 above)
Comment (assafchamoy) [reply src/discrete-layer/components/system-status/cell-renderer/status.cell-renderer.css:48]: OK

File: src/discrete-layer/components/system-status/jobs-dialog.tsx (hunk 5 above)
Comment (assafchamoy) [reply src/discrete-layer/components/system-status/jobs-dialog.tsx:390]: OK

File: src/discrete-layer/components/system-status/jobs-dialog.tsx (hunk 4 above)
Comment (assafchamoy) [reply src/discrete-layer/components/system-status/jobs-dialog.tsx:96]: That is very correct :)

Comment (alebinson) [reply src/discrete-layer/components/system-status/jobs-dialog.tsx:96]: Ok

**Review by alebinson — APPROVED** [review]
Good job

**Pull request merged**